websitev2/docs/user_guide/quickstart.md

```diff
@@ -0,0 +1,278 @@
+
+### Deploy a free version of Unleash to Heroku
+
+[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://www.heroku.com/deploy/?template=https://github.com/Unleash/unleash)
+
+### Deploy a paid version of Unleash to DigitalOcean
+
+> You'll receive 100\$ in free credits if you are a new DigitalOcean user using this link.
+
+[![Deploy to DigitalOcean](https://www.deploytodo.com/do-btn-blue.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/Unleash/unleash/tree/master&refcode=0e1d75187044)
+
+### Accessing your new instance
+
+Once you have set up the new instance, click the URL provided by either Heroku or DigitalOcean and you'll be taken to the application login screen.
+
+Input the following credentials to log in:
+
+```
+username: admin
+password: unleash4all
+```
+
+### Create your first toggle
+
+In order to create a toggle through the UI, [you can follow this guide](create-feature-toggle.md). Once you have created your feature toggle, you are ready to connect your application using an SDK.
+
+If you'd like to create your feature toggles with code, you can hit the create feature endpoint with the following command:
+
+> CRUD operations require an admin API key. For security reasons we have split the admin and client API into separate APIs. You can view how to create API keys in the next section of this guide. Make sure you create client keys for use in SDKs and restrict Admin api key usage.
+
+```curl
+curl -H "Content-Type: application/json" \
+     -H "Authorization: MY-ADMIN-API-KEY" \
+     -X POST \
+     -d '{
+  "name": "my-unique-feature-name",
+  "description": "lorem ipsum..",
+  "type": "release",
+  "enabled": false,
+  "stale": false,
+  "strategies": [
+    {
+      "name": "default",
+      "parameters": {}
+    }
+  ],
+  "variants": [],
+  "tags": []
+}' \
+http://CHANGEME/api/admin/features
+```
+
+### Connect your SDK
+
+Next, find the navigation, open up the Admin panel and find the API Access tab. Click the "Add new api key" button and create a client key. This key can be used to connect to the instance with our [SDKs](sdks/index.md).
+
+You can find more [information about API keys here](token.md).
+
+Now that you have your API key created, you have what you need to connect to the SDK (NodeJS example):
+
+```javascript
+const { initialize } = require('unleash-client');
+const unleash = initialize({
+  url: 'https://your.heroku.instance.com/api/',
+  appName: 'my-app-name',
+  instanceId: 'my-unique-instance-id',
+  customHeaders: {
+    Authorization: 'YOUR_API_KEY_HERE',
+  },
+});
+
+unleash.on('synchronized', () => {
+  // Unleash is ready to serve updated feature toggles.
+
+  // Check a feature flag
+  const isEnabled = unleash.isEnabled('some-toggle');
+
+  // Check the variant
+  const variant = unleash.getVariant('app.ToggleY');
+});
+```
+
+## I want to run Unleash locally
+
+### Run Unleash with docker-compose
```

Comment:
Are we sure that docker-compose is easier than just plain docker these days?

(from [link](https://docs.getunleash.io/deploy/getting_started#option-one---use-docker)):
1. Create a network by running `docker network create unleash`
2. Start a postgres database:
```sh
docker run -e POSTGRES_PASSWORD=some_password \
  -e POSTGRES_USER=unleash_user -e POSTGRES_DB=unleash \
  --network unleash --name postgres postgres
```
3. Start Unleash via docker:
```sh
docker run -p 4242:4242 \
  -e DATABASE_HOST=postgres -e DATABASE_NAME=unleash \
  -e DATABASE_USERNAME=unleash_user -e DATABASE_PASSWORD=some_password \
  -e DATABASE_SSL=false \
  --network unleash unleashorg/unleash-server
```